### ISSUE TITLE
[BITV] 9.4.1.2/3.6 - The "Share" buttons in the table should be implemented as buttons and not as links. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37103]
- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v3-n6

</details>

- [ ] 9.4.1.2/10.6 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v10-n6

**Comment by skjnldsv** [comment]
Will be fixed with F2V for 28 automatically
